tests/firstAutotests.spec.ts

```diff
@@ -1,83 +1,62 @@
 import { test, expect } from '@playwright/test';
-
+import { LoginPage } from '../pages/LoginPage';
+import { ProductPage } from '../pages/ProductPage';
+import { HomePage } from '../pages/HomePage';
 
 test('Verify login with valid credentials', async ({ page }) => {
-  await page.goto('https://practicesoftwaretesting.com/auth/login');
-
-  await page.getByPlaceholder('Your email').fill('customer@practicesoftwaretesting.com');
-  await page.getByPlaceholder('Your password').fill('welcome01');
+  const loginPage = new LoginPage(page);
 
-  await page.click('.btnSubmit');
+  await loginPage.goto();
+  await loginPage.login('customer@practicesoftwaretesting.com', 'welcome01');
 
   await expect(page).toHaveURL('https://practicesoftwaretesting.com/account');
```

Comment:
```suggestion
  await expect(page).toHaveURL('/account');
```
Its better to have relative url here.

Reply:
This part is from the previous lesson, it's already approved